applications/NXem.nxdl.xml

```diff
@@ -0,0 +1,1693 @@
+                    
+                    Additional time stamps prior to preparation_date should better be placed in resources
+                    which describe but do not pollute the description here with prose. Resolving these
+                    connected metadata is considered as within the responsibility of the
+                    research data management system and not the a NeXus file.
+                </doc>
+            </field>
+            <field name="name" type="NX_CHAR" recommended="true">
+                <doc>
+                    An alias used to refer to the specimen to please readability for humans.
+                </doc>
+            </field>
+            <field name="atom_types" type="NX_CHAR">
+                <doc>
+                    List of comma-separated elements from the periodic table that are contained in the sample.
+                    If the sample substance has multiple components, all elements from each component
+                    must be included in atom_types.
+                    
+                    The purpose of the field is to offer research data management systems an opportunity
+                    to parse the relevant elements without having to interpret these from the resources
+                    pointed to by identifier_parent or walk through eventually deeply nested groups in
+                    individual data instances.
+                </doc>
+            </field>
+            <field name="thickness" type="NX_NUMBER" optional="true" units="NX_LENGTH">
+                <doc>
+                    (Measured) sample thickness.
+                    
+                    The information is recorded to qualify if the beam used was likely
+                    able to shine through the specimen. For scanning electron microscopy,
+                    in many cases the specimen is typically thicker than what is
+                    illuminatable by the electron beam.
+                    
+                    In this case the value should be set to the actual thickness of the specimen
+                    viewed for an illumination situation where the nominal surface normal of the
+                    specimen is parallel to the optical axis.
+                </doc>
+            </field>
+            <field name="density" type="NX_NUMBER" optional="true" units="NX_ANY">
+                <doc>
+                    (Measured) density of the specimen.
+                    
+                    For multi-layered specimens this field should only be used to describe
+                    the density of the excited volume. For scanning electron microscopy
+                    the usage of this field is discouraged and instead an instance of a region-of-interest within connection to individual :ref:`NXevent_data_em`
+                    instances can provide a cleaner description of the relevant details
+                    why one may wish to store the density of the specimen.
```

Comment:
Delete the last bit?  'why one may wish...'

Reply:
Removed in the new draft following your suggestion